e2e/atlas/datalake_test.go

```diff
@@ -73,17 +74,17 @@ func TestDatalake(t *testing.T) {
 		resp, err := cmd.CombinedOutput()
 
 		if err != nil {
-			t.Fatalf("unexpected error: %v, resp: %v", err, string(resp))
+			t.Errorf("unexpected error: %v, resp: %v", err, string(resp))
```

Comment:
this one should remain `Fatalf` you won't be able execute the rest of this test if the commands fail.
Like I mention before `Errorf` won't cancel the rest of the test while `Fatalf` will so we use them accordingly, if it's an error the test won't be able to recover or where the next step depends on the previous we go for fatal, if we can keep going (usually the result is different kind of thing) then we use error

Reply:
Ai I read `Errorf` and saw `Fail()` thinking that'd stop the test case but apparently that's different to `FailNow()`. This is not my favourite testing api 😅

Reply:
So `Fatalf` will not only fail the current test but also fail the rest of the tests? It could cause an issue where e.g. the `describe` command fails, but the rest of the commands are fine.

Can we just fail the current subtest?

Reply:
so far it fails in the current block/subtest like you mention, so as soon you get a Fail, no other test inside of `t.Run("Describe", func(t *testing.T) {` will run, but tother blocks will work fine

Reply:
If thats the case, shouldn't we just prefer `Fatalf` everywhere (at least how our tests currently stand where we check only one property of the returned object)?

Reply:
yep, my last comment was about just the last check, that could have been `Errorf` but it being last didn't matter much that's why I didn't block on it, it only matters if someone were to add and extra check after, we have a couple of test that do that

Reply:
Ah ok I get what you mean